yang-module-versioning/draft-ietf-netmod-yang-module-versioning.xml

```diff
@@ -525,84 +528,64 @@ revision 2019-01-02 {
-    </t>
-    <t>If any nbc-change-at extension statements exist in any YANG statements of a module
-    or sub-module, then the module or sub-module MUST have non-backwards-compatible-revision
+    <t>If any nbc-change-at extension statements exists in a module or sub-module, then the module or sub-module MUST have non-backwards-compatible-revision
     substatements in each revision statement of the module or sub-module history where the 
-    revision matches the argument of any nbc-change-at statements. If any revision statements are removed, then all YANG statement compatibiilty extension statements that reference that revision MUST also be removed. Conversely, YANG statement compatibilty extension statements MUST NOT be removed unless the
+    revision matches the argument of any nbc-change-at statements. If any revision statements are removed, then all node compatibiilty statements that reference that revision MUST also be removed. Conversely, node compatibilty statements MUST NOT be removed unless the
     associated revision statement in the revision history is removed.
     </t>
-    <t>If a compatiblity extension is added to a 'grouping' statement,
+    <t>If a node compatiblity statement is added to a grouping,
     then all instances where the grouping is used in the module or
-    by an importing modules are also impacted by the compatibilty information. 
+    by an importing module are also impacted by the compatibilty information. 
     Similarly for a 'typedef', all 
     leafs and list-lists that use that typedef share the specified 
     compatibility classification. A non-backwards-compatible change 
     to a typedef or grouping defined in one module that is used by 
     an importing module, does not cause the importing module to add a
     non-backwards-compatible-revision statement to the revision history.
     Non-backwards-compatible marking does not carry through import statements.
-    This approach avoids having to find and update N importing modules
-    when an imported module has a non-backwards-compatible change.
     </t>
-    <t>YAMG statement compatibility statements are not supported on 
+    <t>A node compatibility statement at a leaf or leaf-list context
+    takes precedence over a node compatibility statement in a typedef
+    used by the leaf or leaf-list. If multiple typedefs with 
+    compatibility statements are used by a leaf or leaf-list (e.g. a union), and there 
+    is no compatibility statement at the leaf or leaf-list context, then
+    the order of precedence used to classify the compatibility of the leaf
+    or leaf-list is as follows: nbc-change-at, bc-change-at and finally editorial-change-at. That is, the leaf or leaf-list takes the most impactful change 
+    classification of all the typedefs.</t>
+    <t>Node compatibility statements are not supported on YANG
```

Comment:
For the last part, focus on the union construct since that can also be used within a typedef. 
use significant instead of impactful

Reply:
done in local draft - will be part of next checkin in this PR.